x-pack/plugins/observability/public/pages/slos/components/grouped_slos/group_list_view.tsx

```diff
@@ -20,6 +32,11 @@ interface Props {
   sort: string;
   direction: SortDirection;
   groupBy: string;
+  summary: {
+    worst?: number;
```

Comment:
Currently we calculate the `worst` value based **only on the violated SLOs**.  If there are no violated SLOs in the group, we don't display anything.

Based on the screenshot, in the `EU region` group below, all SLOs are healthy and we need to display the worst performing SLO of these healthy SLOs in green color. 

![image](https://github.com/elastic/kibana/assets/2852703/e832906b-7085-4433-8e6e-7821a28d7b58)

As we already discussed, we need @maciejforcone's feedback regarding if we need to change the logic behind the worst performing SLO:

```
If there are no violated SLOs, get the worst SLO of the healthy ones and display it in green color

Otherwise get the worst SLO of the violated ones and display it in red
```

Reply:
Hmm, since this UX looks a bit old I'm confirming with @maciejforcone concerning the first point and if we need to have `worst performing` or `worst violated`

Reply:
@dominiqueclarke Actually this is the new UI regarding the group panels. The groupBy dropdown in the screenshot is indeed the old UX